bridge/client/app/_components/ktb-modify-uniform-subscription/ktb-modify-uniform-subscription.component.ts

```diff
@@ -143,18 +199,61 @@ export class KtbModifyUniformSubscriptionComponent {
 
   public updateSubscription(projectName: string, integrationId: string, subscription: UniformSubscription): void {
     this.updating = true;
-    let update;
+    const updates = [];
     subscription.event = `${EventTypes.PREFIX}${this.taskControl.value}.${this.taskSuffixControl.value}`;
     subscription.setIsGlobal(this.isGlobalControl.value, projectName);
 
     if (this.editMode) {
-      update = this.dataService.updateUniformSubscription(integrationId, subscription);
+      updates.push(this.dataService.updateUniformSubscription(integrationId, subscription));
     } else {
-      update = this.dataService.createUniformSubscription(integrationId, subscription);
+      updates.push(this.dataService.createUniformSubscription(integrationId, subscription));
+    }
+
+    if (this.webhookSettings) {
+      const webhookConfig: WebhookConfig = new WebhookConfig();
+      webhookConfig.type = subscription.event;
+      webhookConfig.filter = subscription.filter;
+      webhookConfig.prevConfiguration = this._previousFilter;
+      webhookConfig.method = this.webhookSettings.getFormControl('method').value;
```

Comment:
Also here if we provide an `@Output` for e.g. webhookChanged in the webhook settings, we then have a defined API and there is no need to include it with ViewChild.